english/TheTaleOfTheProgrammer

```diff
@@ -0,0 +1 @@
+once upon a time there was a random 14 year old kid who decided to learn git and github. little did he know that the pulling request bullshit would fuck his brain.
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.